### ISSUE TITLE
pubmed_180601.dtd should be included

### ISSUE THREAD

**Issue by namoshizun** [biopython/biopython#1698]
### Setup

I am reporting a problem with Biopython version 1.71, Python version 2.7.15, and operating
system as follows:

[GCC 4.2.1 Compatible Clang 4.0.1 (tags/RELEASE_401/final)]
CPython
Darwin-15.6.0-x86_64-i386-64bit

### Expected behaviour
Entrez should be able to correctly parse the latest PubMed XMLs complying the latest DTD [pubmed_180601.dtd](http://dtd.nlm.nih.gov/ncbi/pubmed/out/pubmed_180501.dtd). 

### Actual behaviour
Currently still uses pubmed_180101.dtd so the parse sometimes may fail.  

### Steps to reproduce
Example PubMed article: 28895255.
The article's raw abstract XML has the math tag ```<sup>```, which is not supported in the currently used pubmed_180101.dtd. So the abstract text is incomplete. 

```
from Bio import Entrez
handle = Entrez.efetch(
	db='pubmed',
	retmode='xml',
	id='28895255'
)
results = Entrez.read(handle)
print results['PubmedArticle'][0]['MedlineCitation']['Article']['Abstract']['AbstractText']
```

**Comment by mdehoon** [comment]
I believe this has been fixed already. Can you confirm that this error does not occur with Biopython version 1.72 ?

**Comment by namoshizun** [comment]
@mdehoon Yep can confirm this behaviour has been corrected since 1.72. Thanks!
